.travis.yml

```diff
@@ -5,7 +5,4 @@ os:
   - osx
 
 go:
-  - 1.8
-
-install:
-  - go get golang.org/x/sys/unix
+  - 1.13
```

Comment:
Looks like it is better to test things not only against the latest version. Feel free to add the 1.13 or gotip, but 1.8 is also fine I think.

Reply:
I changed it to test against golang stable.

Is there something special about 1.8 that we should add it?